contents/blog/how-to-fix-your-growth-team.md

```diff
@@ -0,0 +1,133 @@
+
+In this post, Alexey shares his advice on how to get your growth team winning again.
+
+TLDR; there are 3 areas to investigate:
+
+1. Your team's velocity.
+2. Your experiment win rate.
+3. Your experiment win size.
+
+## Problem 1: Your team is moving too slowly
+
+Your PM complains that the team is moving too slowly. The engineering team says there is too much technical debt. Sound familiar? This is caused by engineers having the wrong mindset.
+
+Slow velocity is usually a result of code quality being too high. "Engineers hate doing growth. They want to implement elegant solutions to complex problems. Growth engineering is the opposite of that", says Alexey.
+
+### How to fix it
+
+First, sit with the engineering team and prioritize your technical debt. Figure out what is *actually* slowing you down (or likely to) and by how much. Once you have a ranked list, assign 10%-20% of engineering time to clean this up.
+
+Second (and more importantly), urge your team to take shortcuts. The role of a growth engineer is not necessarily to build features, but rather to validate whether a feature should exist. 
+
+With this in mind, a good team exercise is to look at your last 20 experiments and identify which ones you could have validated faster. Thomas Owers, former growth lead at [Let's Do This](https://www.letsdothis.com/), has [good advice](/blog/how-to-start-a-growth-team#thomass-golden-rule-make-your-code-changes-as-small-as-possible) on this:
+
+"Instead of asking 'How long will a project take?', ask 'What can you do in only 1 week? 3 days? Half a day?'. Often, you'll find that you can reduce the scope of the change while still being able to validate your hypothesis".
+
+## Problem 2: Your experiment win rate is low
+
+You keep trying new ideas, but none of your experiments are working. 
+
+"You need better ideas," says Alexey, "and, counter-intuitively, the way to do this is to spend more time adding clear estimates to your current ideas."
+
+Typically, teams prioritize their roadmaps by using frameworks such as [RICE](https://www.intercom.com/blog/rice-simple-prioritization-for-product-managers/) and assigning t-shirt size estimates ("S", "M", "L"). This doesn't go deep enough and leads to teams wasting time by building pointless things. 
+
+Spending only five minutes estimating something your team will spend weeks on doesn't make sense.
+
+### How to fix it 
+
+Once you have shortlisted the ideas you want to work on, the next step is to estimate the dollar impact of each one. Because you'll know **exactly** what to expect from your experiments, the result is a higher win rate.
+
+This approach was made famous by Facebook Messenger's VP of growth Darius Contractor and his [EVELYN framework](https://www.drift.com/blog/secret-to-faster-growth/) (He even shares a handy [sheet with real examples you can copy](https://airtable.com/appzs6HT2ggrJt9pN/shrInOxl0SL1Xmauk/tblTQFEMcXAaEgSpM/viwZessbXpiA8q2jd?blocks=bipMQFWTjqu0HI3wv)).
```

Comment:
Can you give a little about the EVELYN framework here?

It also isn't immediately obvious how changes to Facebook Messenger have a dollar impact? I've never seen an ad there (although I know there is connections to the business). Maybe clarify what an active user is worth?

Reply:
Actually, maybe you should remove this paragraph because you repeat it down below.

Reply:
reworked